text/0000-icons.md

```diff
@@ -0,0 +1,137 @@
+- Start Date: 2021-03-14
+- RFC PR:
+- TVA Issue:
+
+# Summary
+
+Bundle icons as inline SVG into a simple, organized structure, ready for import and use in an application.
+
+Icons are formatted and optimized by automated build scripts.
+
+By default, the icon's color is inherited from the `color` property of the parent element.
+
+# Basic example
+
+## React (JSX)
+
+Import SVG as JSX element (default) from the library and use in your application.
+
+```jsx
+import { bookmark } from '@pluralsight/icons'
```

Comment:
This example doesn't reflect the #detailed-design section that you have. Please make sure these are consistent. Let's also remove the style Object syntax as we want to promote using headless-styles with it.

Reply:
I can remove the style object.  I didn't want to use the headless-styles since that hadn't been defined yet for icons, so I used the styles to also show an example some of the style attributes that might be returned by that process.  Other than that I'm not sure what else is missing from the detailed design.  Should I go into detail about the build process?  I also didn't do anything with the directory structure, since that hadn't been defined yet.  Are we defining that here as well?